Sources/JWTProvider/PayloadAuthenticationMiddleware.swift

```diff
@@ -8,19 +8,46 @@ import AuthProvider
 /// Parses JWT and creates an ephemeral session,
 /// logging the user in with credentials from the token.
 public final class PayloadAuthenticationMiddleware<U: PayloadAuthenticatable>: Middleware {
-    let signer: Signer
+    internal private(set) var signers: SignerMap
     let claims: [Claim]
+    let jwksURL: String?
+    let clientFactory: ClientFactoryProtocol?
 
     /// Create a LoginMiddleware specifying
-    /// the JWT signer and type of payload
+    /// the JWT signers and type of payload
     /// that will be stored in the JWT
     public init(
         _ signer: Signer,
         _ claims: [Claim] = [],
         _ userType: U.Type = U.self
+        ) {
```

Comment:
I know, it's annoying! I think the consensus is that we sometimes ignore Xcode when it comes to formatting.